wolox-books/src/sass/styles.scss

```diff
@@ -0,0 +1,83 @@
+* {
+  font-family: 'Roboto', sans-serif;
+  background: #F4F4F4;
+}
+
+.navbar{
+  border-top: 5px solid $blue;
+  padding: 40px;
+  text-align: center;
+
+  .logo {
+    max-width: 500px;
+    display: block;
+    margin: 0 auto;
+  }
+
+  .subtitle{
+    font-weight: bold;
+    letter-spacing: 8px;
+    font-size: 1em;
```

Comment:
we'll delete this when I send you the image but just for you to know, we tend to use pixels for our layouts since designers create the designs without stablishing an em criteria for texts